src/components/BasicHead.vue

```diff
@@ -233,7 +233,7 @@ export default {
       }
     },
     filterIcon (){
-      return true // this.$store.state.vxg.cmp.BasicHead.show.filterIcon
+      return this.$store.state.vxg.cmp.BasicHead.show.filterIcon
```

Comment:
rename this to filter - filterIcon is too specific - it may not be an icon in future

Reply:
Done